runner/app/pipelines/audio_to_text.py

```diff
@@ -77,11 +76,19 @@ def __call__(self, audio: UploadFile, **kwargs) -> List[File]:
             converted_bytes = audio_converter.convert(audio, "mp3")
             audio_converter.write_bytes_to_file(converted_bytes, audio)
 
+        if kwargs["return_timestamps"] != "false":
+            kwargs["return_timestamps"] = kwargs["return_timestamps"] if kwargs["return_timestamps"] == "word" else True
+        else:
+            kwargs.pop("return_timestamps", None)
+
         try:
             outputs = self.tm(audio.file.read(), **kwargs)
         except Exception as e:
             raise InferenceError(original_exception=e)
 
+        # When timestamps are not requested, add the chunks field to the response to satisfy the response schema
+        if "chunks" not in outputs:
```

Comment:
I think you could use `s.pop("return_timestamps", None)`.

Reply:
This is related to the output. When `return_timestamps` is not used, the `chunks` property is not in the output (ex: `{'text': ""}`). This logic appends the chunks property to satisfy the response schema in TextResponse.

Reply:
@eliteprox I took a look at the behavior, and I believe it's best to keep it consistent with the current model, even though the interface has double typing. I've made these adjustments and merged the pull request. Thanks for the great addition! 🚀

Reply:
Nice improvement with `Union[str, bool]`, I didn't know there was a way to accept two different types in one request parameter